devnets/controverse/assetlist.json

```
@@ -0,0 +1,62 @@
+{
+  "$schema": "../../assetlist.schema.json",
+  "chain_name": "miniwasm",
+  "assets": [
+    {
+      "description": "The native token of Initia",
+      "denom_units": [
+        {
+          "denom": "l2/771d639f30fbe45e3fbca954ffbe2fcc26f915f5513c67a4a2d0bc1d635bdefd",
```

the op bridged init denom is `l2/2716295e82bab3b0308dc1e35837fe471657ea07ee71aebb6129739fb0f1278a` the eaiest why to get it is bridged init with op bridge and check balance (ex https://maze-rest-493360ba-cd86-4627-aca3-8b61635eb2bf.ue1-prod.newmetric.xyz/cosmos/bank/v1beta1/balances/init13rqpwycyhepvxgfmzucat5t6v0veftnf684ua6)

---

So you are saying:
```suggestion
          "denom": "l2/2716295e82bab3b0308dc1e35837fe471657ea07ee71aebb6129739fb0f1278a",
```
?

---

right and please also change the one in chain.json